app/views/warehouse_reports/hmis_exports/_reports.haml

```diff
@@ -18,3 +18,7 @@
     %td
       = link_to warehouse_reports_hmis_export_path(report), method: :delete, class: 'btn btn-sm btn-danger btn-icon-only' do
         %i.icon-cross
+    %td
+      - if recurring? report
```

Comment:
This will need some permissions (you should only be able to cancel ones you start, or with an administrative level permission).